scapy/layers/tls/session.py

```diff
@@ -894,15 +945,25 @@ def __init__(self, _pkt="", post_transform=None, _internal=0,
                 self.tls_session.ipdst = tcp.underlayer.dst
             except AttributeError:
                 pass
+
+            # Load a NSS Key Log file
+            if getattr(conf, "tls_nss_filename", False):
```

Comment:
I am a still skeptical regarding these `getattr(conf, ...)` and I am welcoming feedbacks.

The TLS layer is not being loaded default, I believe that it is better no to add these configuration elements to `scapy/config.py`.